R/add_risktable.R

```diff
@@ -69,9 +69,16 @@ add_risktable.ggsurvfit <- function(
   ,label = NULL
   ,group = "strata"
   ,collapse = FALSE
+  ,rowgutter = .3
```

Comment:
I use 0.16 as not to introduce changes for users that have implemented previous version + needs to be consistent with documentation.